Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    Dutchie-Style (1.0.4)
+    Dutchie-Style (1.0.8)
```

Comment:
These changed automatically when I changed the version, I didn't update the gems or anything. I am sort of unsure why it did it now and not before, but the changes here also seem fine.